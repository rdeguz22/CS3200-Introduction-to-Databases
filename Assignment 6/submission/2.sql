SELECT d.first_name || ' ' || d.last_name AS DIRECTOR,
       COUNT(director_id) AS MOVIES,
       MIN(rank) AS MIN_RANK,
       AVG(rank) AS AVG_RANK,
       MAX(rank) AS MAX_RANK
    FROM DIRECTORS AS d
        INNER JOIN movies_directors AS md ON d.id = md.director_id
        INNER JOIN movies AS m ON md.movie_id = m.id
    WHERE rank IS NOT NULL
    GROUP BY d.first_name, d.last_name
    HAVING 2 <= COUNT(director_id) AND 7 <= MIN(rank)