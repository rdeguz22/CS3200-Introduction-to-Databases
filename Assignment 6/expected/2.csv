DIRECTOR,MOVIES,MIN_RANK,AVG_RANK,MAX_RANK
Ethan Coen,2,7.8,8.0,8.2
Joel Coen,2,7.8,8.0,8.2
Quentin Tarantino,4,8.2,8.4,8.7
